src/main/java/wooteco/subway/web/ExceptionAdvice.java

```diff
@@ -0,0 +1,72 @@
+package wooteco.subway.web;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import wooteco.subway.service.favorite.ExistedFavoriteException;
+import wooteco.subway.service.member.ExistedEmailException;
+import wooteco.subway.service.member.NotExistedEmailException;
+import wooteco.subway.service.member.WrongPasswordException;
+import wooteco.subway.service.path.DuplicatedStationException;
+import wooteco.subway.service.path.NotExistedPathException;
+import wooteco.subway.service.path.NotExistedStationException;
+import wooteco.subway.web.member.InvalidAuthenticationException;
+
+@RestControllerAdvice
+public class ExceptionAdvice {
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(DuplicatedStationException.class)
+    public ExceptionResponse handleDuplicatedStation() {
+        return new ExceptionResponse("DUPLICATED_STATION");
```

Comment:
DuplicatedStationException에서 설정한 에러메시지가 전달되지 않겠네요
```suggestion
    public ExceptionResponse handleDuplicatedStation(DuplicatedStationException e) {
```
예외를 인자로 받아서 메시지를 직접 뿌리면 어떨까요

Reply:
그리고 HttpStatus가 동일한 케이스는 하나로 모아보면 어떨까요
RuntimeException으로 받을 수 있는지 한 번 테스트 해보시면 좋겠네요
```java
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    @ExceptionHandler(value = { DuplicatedStationException.class, NotExistedPathException.class })
    public ExceptionResponse handleDuplicatedStation(RuntimeException e) {
        return new ExceptionResponse(e.getMessage());
    }
```

Reply:
프론트에서 특정이름을 error sign이나 code를 받기 위해서 advice에서 넣어서 보내주고 있는데요!
한꺼번에 예외를 잡는다면, 각각의 커스텀예외 클래스안에서 코드를 가지고있어야할까요??

Reply:
만약 보여주신 샘플대로, 한번에 잡는다면
프론트에서 에러 응답을 받았을때, 각각의 에러를 구별하는 방법이 궁금합니다.

만약 메세지로 구별해야한다면, 프론트에서 사용할 "DUPLICATED_STATION"과 같은 상수를 메세지로 보내는게 좋을까요??

Reply:
-----
우선은 에러코드를 담을 수 있는 공통 커스텀 예외를 정의해보았습니다!
혹시 더 좋은 방법이 있다면 조언 부탁드립니다.!!

Reply:
프론트에서는 http status code 값에 따라 구별하거나 특정 상황에 맞게 다른 메시지로 바꾸면 됩니다.
제가 의도한 대로 잘 수정해주셨네요 👍